### ISSUE TITLE
AV-8B Fuel Panel Total Quantity displays 4 digits of fuel quantity preceeded by 0

### ISSUE THREAD

**Issue by GAKB** [HeliosVirtualCockpit/Helios#604]
Instead of displaying a 5 digit readout of fuel quantity e.g. "11700" the Fuel Panel Total Quantity drum displays 4 of the digits preceeded by a 0 e.g. "01700".

This also happens with a separate number display configured to show Total Quantity.

[Helios_status_report_133056680616707154.txt](https://github.com/HeliosVirtualCockpit/Helios/files/9396849/Helios_status_report_133056680616707154.txt)

**Comment by BlueFinBima** [comment]
@GAKB Yes another good spot.  Never loaded a Harrier up with that much fuel before;-)

**Comment by GAKB** [comment]
Thanks!
The Harrier is my only module so I've been pushing it hard 😋

**Comment by BlueFinBima** [comment]
No problem.  If you want to fix this one before the official fix comes out, then change line 427 in AV8BNA.lua in '..\Saved Games\DCS.openBeta\scripts\Helios\drivers\`
to 
```
    helios.send(2010, string.format("%.0f",mainPanelDevice:get_argument_value(366) * 100000+mainPanelDevice:get_argument_value(367) * 10000+mainPanelDevice:get_argument_value(368) * 1000+mainPanelDevice:get_argument_value(369) * 100+mainPanelDevice:get_argument_value(370) * 10))    -- Fuel Total
```
This one is fixed in commit https://github.com/HeliosVirtualCockpit/Helios/pull/610/commits/504fca8fae7facfa8d0b4c5ad434f34a4aa40901

**Comment by GAKB** [comment]
That's great! Thanks for this.

**Comment by BlueFinBima** [comment]
Shipped in [1.6.5307.9](https://github.com/HeliosVirtualCockpit/Helios/releases/latest)
